crates/ui/src/components/title_bar/windows_window_controls.rs

```diff
@@ -98,9 +98,9 @@ impl WindowsCaptionButton {
 
 impl RenderOnce for WindowsCaptionButton {
     fn render(self, _cx: &mut WindowContext) -> impl IntoElement {
-        // todo(windows): get padding from win32 api, need HWND from window context somehow
-        // should be GetSystemMetricsForDpi(SM_CXSIZE, dpi)
-        let width = px(36.0);
+        // todo(windows) report this width to the Windows platform API
+        // NOTE: this is intentionally hard coded. An option to use the 'native' size
```

Comment:
Could this comment be clarified?

Reply:
looks like I didn't finish my sentence. my bad 😅